src/server_manager/web_app/ui_components/outline-server-list.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright 2021 The Outline Authors
```

Comment:
@mpmcroy this is another case where I think Vini would run into the same problem I did with the dialog if he wanted to write unit tests for this element.  Let's put our heads together and figure this out so we can increase our test coverage as we migrate the UI to TS / LitElement?